source/Willow-Bootstrap/WillowBootstrapFileMetadataLibrary.class.st

```diff
@@ -40,8 +40,8 @@ WillowBootstrapFileMetadataLibrary >> cleanModalFunctionUsing: script [
 	function := JSFunction new arguments: #(modal).
 	function << ((script jQuery new alias: 'modal') next: WAScriptTag new tag) remove.
 	function << (script jQuery new alias: 'modal') remove.
-	function << ((script jQuery id: WillowConstants dialogSectionName) children length equalTo: 0)
-		nonInlinedThen: (script jQuery class: BootstrapCssStyles modalBackdrop) remove.
+	function << ((script jQuery id: script constants >> #willow >> #dialogSectionName) children length equalTo: 0)
```

Comment:
If possible, it would be better to rewrite this method to avoid having the section name accessed through the constants provided by the script, and then the classification by calling the Willow namespace. We should have a way to do this more consistently.